docs/developers/optimistic-oracle-integration.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Integrating with the Optimistic Oracle
+sidebar_label: Integrating with the Optimistic Oracle
+---
+
+This demo will set up an `OptimisticDepositBox` contract which custodies a user’s ERC-20 token balance.
+
+On a local testnet blockchain, the user will deposit wETH (Wrapped Ether) into the contract and withdraw wETH denominated in USD. For example, if the user wants to withdraw $10,000 USD of wETH, and the ETH/USD exchange rate is $2,000, they would withdraw 5 wETH.
+
+* The user links the `OptimisticDepositBox` with one of the price identifiers enabled on the DVM.
+
+* The user deposits wETH into the `OptimisticDepositBox` and register it with the `ETH/USD` price identifier.
+
+* The user can now withdraw a USD-denominated amount of wETH from their `DepositBox` via smart contract calls, with the Optimistic Oracle enabling optimistic on-chain pricing.
```

Comment:
nit: feel like "on-chain pricing" is a little misleading. Maybe something like: OO enabling off-chain prices to be pulled on-chain.

Reply:
I'm not sure I agree, since this lets you set prices optimistically and only pull prices on-chain if there is a dispute. I would argue that the DVM allows you to pull prices on-chain and the optimistic pricing is the new thing provided by the OO.

Reply:
Fair - good distinction.